components/com_joomgallery/models/edit.php

```diff
@@ -800,4 +810,4 @@ public function publish()
 
     return true;
   }
-}
\ No newline at end of file
+}
```

Comment:
code style: not sure here, @Chraneco: do we add a new line at the end of a file now?